transformations/migration/index.md

```diff
@@ -39,3 +39,15 @@ All transformations that meet the following conditions will be migrated. They mu
 The following transformations will be skipped over in the migration process. They will not be migrated:
 - Disabled transformations in the bucket 
 - Transformations that have both multiple [phases](/transformations/#phases) and multiple transformation [backends](/transformations/#backends); in this case, you must split the bucket, so that each of the resulting buckets have either one phase or one backend. If you are not sure how to do that, please contact our support.
+
+## What happens during migration:
+- We do check automatically if the transformation is eligible for migration
```

Comment:
za mna ok, ale po jazykovej stranke by to mal niekto skontrolovat .... podla mna tu chybaju `.` za vetami.
... a tato veta by mohla vyt skor `We automatically check if the transformation is suitable for migration.` ... ale `EN` nie je moja silna stranka.

Reply:
je to zkopírovaný od Tomise, ale přidám ještě @hhanova na kontrolu :)

Reply:
ok, tak sa spytaj Tomise, ci to preslo jazykovou korekturou, a ak ano tak kludne dam approve.

Reply:
Myslim, ze tie bodky by tam mali byt, .... minimalne je to tak vyssie v texte.

Reply:
Na sprint retro sme riesili, ze niektore texty od produktu su OK, a niektore nie, ... ze sa s tym bude nieco musiet robot.